admin_guide/manage_authorization_policy.adoc

```diff
@@ -0,0 +1,283 @@
+= Managing Authorization Policy
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+You can use the administrator CLI to manage
```

Comment:
@adellape The "administrator CLI" term is used a lot in this PR. I'm not sure what that is. It it just the OpenShift CLI tools? If so, then "administrator" could probably be taken away. If it's different, is there somewhere in the docs where the difference is explained? Maybe in the CLI reference section.